README.md

```diff
@@ -1 +1,35 @@
-# cvs-wms
\ No newline at end of file
+# cvs-wms
+
+Create WMS database locally
+
+## Prerequisites :
+installed Liquibase
+installed MySQL Database Software
+installed MySQL Connector/J (https://dev.mysql.com/doc/connectors/en/connector-j-binary-installation.html)
+
+## Liquibase Steps
+
+### Create configuration file liquibase.properties, provide database name (default: WMS_EDH_STAGING), database username and password and class path to MySQL J/Connector.
+
+```properties
+driver: com.mysql.cj.jdbc.Driver
+url: jdbc:mysql://localhost:3306/CVSBNOP?createDatabaseIfNotExist=true
```

Comment:
The example refers to `CVSNOP`, potentially needs to be corrected to `EDH_WMS_STAGING` to better reflect the use case.